src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -82,7 +89,14 @@ def get_device_by_args(args: list):
             break
     return device
 
-# Class to read test cache    
+def get_suite_filter(test_filter: str, suite_filter : str) :
+    filters = test_filter.strip('\"').split('*')
+    for filter in filters :
+        if (filter and suite_filter.find(filter) == -1) :
+            suite_filter += f'*{filter}'
```

Comment:
I don't see any combinantion of `--gtest_filters` and suite_name

Reply:
it is not required yet

Reply:
Minor comment, looks like we can mark this method as static and move to Runner class

Reply:
added support of patterns combinations in advance

Reply:
don't see any benefits to make it static function of the class but moved, please verify